ynr/apps/uk_results/models.py

```diff
@@ -70,7 +70,7 @@ def as_dict(self):
             data["candidate_results"].append(
                 {
                     "num_ballots": result.num_ballots,
-                    "is_winner": result.is_winner,
+                    "elected": result.membership.elected,
```

Comment:
I wasn't entirely sure how necessary this change was, so advice welcome - it might be that key/value can be removed as the `is_winner` field is being removed from the model, or that we want to keep it as `is_winner` rather than elected?

As part of this, I wrote the data migration above which will go through every version stored in the DB and change the key from `is_winner` to `elected`. Again wasn't 100% about this change, as it is rewriting history somewhat. But otherwise we would have inconsistent versions

Reply:
I think this is reasonable. 

FWIW the reason for the name change was that "winner" implies there was a single person who won, whereas "elected" is slightly better when >1 seat is contested.